### PR TITLE
template-deprecation: add the presubmit

### PR DESCRIPTION
**Pull request by petr-muller**
/hold

Needs the image to exist (https://github.com/openshift/release/pull/13418) and also maybe adding the tool to autoconfigbrancher

### CODE REVIEW

File: core-services/template-deprecation/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+approvers:
+    petr-muller
+reviewers:
+    petr-muller
```
Comment (petr-muller) [inline core-services/template-deprecation/OWNERS:4]: My intention behind this to be (for a short while) a bottleneck for these PRs, to understand the volume with which the allowlist will change in day-to-day operation. Any approver from the root OWNERS will still have the power to approve, so it should not be a real probllem

**Comment by petr-muller** [comment]
We need diff: https://github.com/openshift/ci-tools/pull/1390

**Comment by stevekuznetsov** [comment]
why is broken?

**Comment by petr-muller** [comment]
> why is broken?

It's not, the job is doing exactly what it should. Some [new jobs were merged](https://github.com/openshift/release/pull/13436) since I generated the allowlist in this PR. The current rehearsal shows how the presubmit would have failed on Ben's PR that added them.

**Comment by petr-muller** [comment]
Rebased, used `make template-allowlist` to regenerate the allowlist, discovered a problem, fixed the problem and improved the error message.

**Review by alvaroaleman — APPROVED** [review]

**Review by droslean — APPROVED** [review]

**Pull request merged**